keyboards/redscarf_iiplus/verd/readme.md

```diff
@@ -0,0 +1,24 @@
+# Red Scarf II+ Ver.D (RS78)
+
+A 60% keyboard with a numpad on the right
+
+* Keyboard Maintainer: [Andy Lee](https://github.com/alittlepeace)
+* Hardware Supported: Ver.D (RS77)
+* Hardware Availability: [Drop](https://drop.com/buy/red-scarf-ii-plus-ver-d-custom-mechanical-keyboard-kit)
+
+
+Please note that QMK Firmware does not currently support using the remote to control the underglow LEDs. 
```

Comment:
```suggestion
Please note that QMK Firmware does not currently support controlling the underglow LEDs; however, if your board came with the remote, it can still be used to change the colour.
```